public/styles/style.css

```diff
@@ -201,9 +196,28 @@ body:before {
     filter: blur(5px);
 }
 
+body {
+    font-family: 'Roboto', sans-serif;
+    text-align: center;
+    color: white;
+}
+
 .team-name input {
     width: 200px;
 }
+
+#team-name-heading {
+    height: 60px;
+    width: 100%;
+    text-align: center;
```

Comment:
Think about combining text-aligns to reduce duplication. 

```
body, 
#team-name-heading {
     text-align: center;
}
```

Reply:
Or, better even, add a class that you can ad in markup that will allow you to text align things, ex:

```
.center-text {
    text-align: center;
}
```

`<h1 class="center-text"></h1>`

Thoughts?

Reply:
Working on it!